packages/notification-services-controller/src/NotificationServicesPushController/NotificationServicesPushController.ts

```diff
@@ -69,6 +69,9 @@ export type NotificationServicesPushControllerMessenger =
     AllowedEvents['type']
   >;
 
+export const defaultState: NotificationServicesPushControllerState = {
```

Comment:
Mobile ask - we are now exporting default exports (this ensures that our redux initial state has "something").